en/simulation/multi_vehicle_simulation_gazebo.md

```diff
@@ -46,6 +49,56 @@ https://youtu.be/aEzFKPMEfjc
 https://youtu.be/lAjjTFFZebI
 {% endyoutube %}
 
+
+
+## Multiple Vehicles with Gazebo based on RTPS {#with_rtps}
```

Comment:
@hamishwillee As you mentioned, this is not actually a different way to run multivehicle simulation in ROS, it is using a method identical that is used in the previous section.

I think @stmoon is trying to explain how to simulate multivehicle simulation with the exposure of the RTPS interface not to explain an additional method of doing multivehicle simulation. Therefore I suggest we move this section to a page explaining how to test RTPS. This page is explaining different methods of simulating multiple vehicles.

Reply:
@Jaeyoung-Lim To me this _looked_ like a different way to run the simulation - I just wasn't sure why this method was needed. So I mentioned nothing :-)

Can you work with @stmoon to make this "correct"?

Reply:
@hamishwillee To communicate with companion boards in multiple vehicles, the RTPS may be used instead of Mavlink. Especially, I think many ROS users are familiar with  RTPS.  Therefore, I think the multivehicle simulation with RTPS is necessary to connect the ROS ecosystem.

Reply:
@stmoon In that case, could you move it to a different page?

Reply:
@Jaeyoung-Lim  I think it is one of way for "Multi-Vehicle Simulation with Gazebo". In addition, it is not enough to allocate different pages for only this description.

Reply:
@stmoon But this is a special case of https://dev.px4.io/master/en/simulation/multi_vehicle_simulation_gazebo.html#no_ros and is not a seperate way of running the multivehicle sim. I am not asking to add a new page for this section, but to add it to a more relevant one (e.g. page explaining RTPS)

Reply:
@hamishwillee Sorry for the late reply.

Actually, I have confused about what I do. 

Do you want to move this explanation to a different existing page or a new page? (For example, https://dev.px4.io/master/en/middleware/micrortps.html ) 
If yes, I think that a new page for simulation is needed. However, I think the explanation should be here because it is one of them for multi-vehicle part
 
OR... 

Do you think that it is a special case of the existing part ("Multiple Vehicles with ROS and Gazebo" or "Multiple Vehicle with Gazebo (No ROS)")? If yes, I will insert the explanation to the existing part.
However, I think it is a different way because it can be used with or without ROS.

Reply:
@Jaeyoung-Lim So I see your point now. The heading "Multiple Vehicles with Gazebo based on RTPS {#with_rtps}" is at the same level as "Multiple Vehicle with Gazebo (No ROS) {#no_ros}" and "Multiple Vehicles with ROS and Gazebo {#with_ros}" so it looks like a "peer" of those.
However actually it is a specific example of the "no ros" case.

I actually don't think moving this to RTPS page would necessarily help here, though we might link from one. 

How about we change so that this is clearly part of "no ros" , moving the intro stuff under "### Build and Test".

So it might looks like this:


### Build and Test (RTPS)

To simulate multiple vehicles in Gazebo without ROS, use the `gazebo_sitl_multiple_run.sh` command in the terminal from the root of the *Firmware* tree (as described above). 
Here we will use the `-t px4_sitl_rtps` option which RTPS for communicating with the PX4 rather than the MAVLink Simulation API **[@Jaeyoung-Lim confirm!]

> **Note** You will need to have installed RTPS and the `micrortps_agent` should be run in the different terminal for each vehicle.
  For more information see: [RTPS/ROS2 Interface: PX4-FastRTPS Bridge](../middleware/micrortps.md).

To build an example setup, follow the steps below:

1. Clone the PX4/Firmware code, then build the SITL code:
   ```bash
   cd Firmware_clone
   git submodule update --init --recursive
   DONT_RUN=1 make px4_sitl_rtps gazebo
   ```

1. build `micrortps_agent`
   * To use agent in ROS-independent RTPS, follow the [installation instructions here](../middleware/micrortps.md#agent-in-a-ros-independent-offboard-fast-rtps-interface)
   * To use the agent in ROS2, follow the [instructions here](../middleware/micrortps.md#agent-interfacing-with-a-ros2-middleware)

1. Run `gazebo_sitl_multiple_run.sh`.
   For example, to spawn 4 vehicles, run:

   ```bash
   ./Tools/gabo_sitl_multiple_run.sh  -m iris_rtps -t px4_sitl_rtps -n 4
   ```

   >  **Note** Each vehicle instance is allocated a unique MAVLink system id (1, 2, 3, etc.), can receive data from a unique remote UDP port (2019, 2021, 2023, etc.), and transmit data to UDP port (2020, 2022, 2024, etc.).
1. Run `micrortps_agent`.
   For example, to connect 4 vehicles, run:

   ```bash
   micrortps_agent -t UDP -r 2020 -s 2019 &
   micrortps_agent -t UDP -r 2022 -s 2021 &
   micrortps_agent -t UDP -r 2024 -s 2023 &
   micrortps_agent -t UDP -r 2026 -s 2025 &
   ```

Reply:
Hi @Jaeyoung-Lim  Could you please check it before modification?. 
I hope to check it by you even though it is modified by me. :-)

Reply:
@stmoon Could you clarify on what you need me to check?

Reply:
@hamishwillee Is there some issues related to @Jaeyoung-Lim? (because @Jaeyoung-Lim  is mentioned in the comment of you) 
If not, I will modify it as you comment.